--- cmake_include/FindHDF4.cmake.orig	Wed Oct 14 15:09:38 2015
+++ cmake_include/FindHDF4.cmake	Wed Oct 14 15:09:15 2015
@@ -33,10 +33,10 @@
 if (WIN32)
 #    set(HDF4_SEARCH_DEFAULT "C:/InstallKits/HDF4.2.6_win_x64")
 	set(HDF4_SEARCH_DEFAULT "C:/Program Files/HDF Group/HDF4/4.2.7")
-	set(HD_NAMES hdf hdfdll hd427m hd426m hd425m )
-	set(HM_NAMES mfhdf mfhdfdll hm427m hm426m hm425m )
+	set(HD_NAMES libhdf hdfdll hd427m hd426m hd425m )
+	set(HM_NAMES libmfhdf mfhdfdll hm427m hm426m hm425m )
 	set(JPEG_NAMES libjpeg jpeg )
-	set(XDR_NAMES xdr )
+	set(XDR_NAMES libxdr )
 	find_library(HDF4_HD_LIBRARY NAMES ${HD_NAMES} HINTS ${HDF4_SEARCH} ENV HDF4_ROOT PATHS ${HDF4_SEARCH_DEFAULT} PATH_SUFFIXES dll bin lib DOC "location of hd dll" NO_SYSTEM_ENVIRONMENT_PATH)
 	find_library(HDF4_HM_LIBRARY NAMES ${HM_NAMES} HINTS ${HDF4_SEARCH} ENV HDF4_ROOT PATHS ${HDF4_SEARCH_DEFAULT} PATH_SUFFIXES dll bin lib DOC "location of hm dll" NO_SYSTEM_ENVIRONMENT_PATH)
 	find_library(JPEG_LIBRARY NAMES ${JPEG_NAMES} HINTS ${HDF4_SEARCH} ENV HDF4_ROOT PATHS ${HDF4_SEARCH_DEFAULT} PATH_SUFFIXES dll bin lib DOC "location of jpeg dll" NO_SYSTEM_ENVIRONMENT_PATH)
