--- setup_build.py.orig	2016-05-10 16:24:25.277820300 +0100
+++ setup_build.py	2016-05-10 16:05:22.621566100 +0100
@@ -38,7 +38,7 @@
 
 if sys.platform.startswith('win'):
     COMPILER_SETTINGS = {
-        'libraries'     : ['h5py_hdf5', 'h5py_hdf5_hl'],
+        'libraries'     : ['hdf5', 'hdf5_hl'],
         'include_dirs'  : [localpath('lzf'), localpath('windows')],
         'library_dirs'  : [],
         'define_macros' : [('H5_USE_16_API', None), ('_HDF5USEDLL_', None)] }
