--- bindings/cpp/NeXusFile.hpp.orig	2019-12-20 15:21:10.850941500 +0000
+++ bindings/cpp/NeXusFile.hpp	2019-12-20 15:21:20.617444500 +0000
@@ -116,7 +116,7 @@
      * This is a deprecated function.
      * \param com The compression type.
      */
-    void compress(NXcompression comp) NEXUS_DEPRECATED_FUNCTION();
+    void compress(NXcompression comp) NEXUS_DEPRECATED_FUNCTION("");
 
     /**
      * Initialize the pending group search to start again.
