--- config/cmake_ext_mod/HDFTests.c.orig	Wed Nov 28 13:12:31 2018
+++ config/cmake_ext_mod/HDFTests.c	Wed Nov 28 13:24:16 2018
@@ -20,11 +20,11 @@
 #include <stdlib.h>
 #include <string.h>
 
 int main(void)
 {
-  char *llwidthArgs[] = { "l64", "l", "L", "q", "ll", NULL };
+  char *llwidthArgs[] = { "I64", "l64", "l", "L", "q", "ll", NULL };
   char *s = malloc(128);
   char **currentArg = NULL;
   LL_TYPE x = (LL_TYPE)1048576 * (LL_TYPE)1048576;
   for (currentArg = llwidthArgs; *currentArg != NULL; currentArg++)
     {
